### PR TITLE
DOCS: lightspeed login text

### PR DESCRIPTION
**Pull request by Lars-MultiSafepay**
### Basic
- We only accept documentation that is proved to work on our LIVE API
- At least 1 approval is needed before pull requests can be merged
- The article must match our Synonym word list. For external commits, a MultiSafepay employee will check
- US English must be applied
- Use bullets for numerations where the sort order doesn't matter. For chronological order, use numbers. This does not apply to changelogs
- Set link on email address.

### Text style
- Paths are written like: _Configuration → System → Plugin → MultiSafepay_
- Use single quotation marks in your Markdown code (example title: 'Title of the article')
- Use Italics for buttons, clicks, paths, etc. Do mind to close the italics before the end of a sentence followed by a period
- Numerations / bullets are punctuated with a period at the end of the numerations / bullets.
- Periods should be excluded for sentences that finish in an e-mail address (e.g. "..contact us at <example@example.com> "  )

### Images
- Screenshot recomended sizes: Width: 820px - Height: auto

### CODE REVIEW

File: content/integrations/hosted/lightspeed_app/faq/how-to-login.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title : "How to login"
+meta_title: "Lightspeed plugin - Can I change the API key after installation? - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+---
+
+In early 2021 we have changed how our login system works for the MultiSafepay Payment app for lightspeed eCom. Our old system was password based, our new system uses the Single Sign On provided by lightspeed. This means you can directly login via Lightspeed eCom store to our settings page of the app.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/how-to-login.md:7]: In early 2021, we have changed how our login system works for the MultiSafepay Payment app for Lightspeed eCom. Our old system was password based, although now our new system uses the Single Sign On provided by Lightspeed. This means you can directly login via Lightspeed eCom store to our settings page of the app.

File: content/integrations/hosted/lightspeed_app/faq/how-to-login.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+To login using the new system you have to use our app store page. Login to the /admin section of your lightspeed store. Go to apps, my apps / "aangeschafte apps" and click on the name of our app, MultiSafepay Payments. This should open the app store page. Alternatively paste "/admin/store/apps/1517" after the base url your store.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/how-to-login.md:9]: To login using the new system you have to use our app store page. Login to the /admin section of your Lightspeed store. Go to apps, my apps / "aangeschafte apps" and click on the name of our app, MultiSafepay Payments. This should open the app store page. Alternatively paste "/admin/store/apps/1517" after the base url your store.

File: content/integrations/hosted/lightspeed_app/faq/how-to-login.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+On the store page click on "Goto app" / "Ga naar app" And you will be able to access your settings page.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/how-to-login.md:11]: On the store page click on "Goto app" / "Ga naar app" and you will be able to access your settings page.

**Review by Atiq-MultiSafepay — CHANGES_REQUESTED** [review]
@Lars-MultiSafepay I've added a few corrections to the text.
Also, I believe it may be better to explain the instructions step-by-step. Let me know your thoughts.

**Comment by Lars-MultiSafepay** [comment]
@Jasper-MultiSafepay added images

File: content/integrations/hosted/lightspeed_app/faq/how-to-login.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title : "How to login"
+meta_title: "Lightspeed plugin - Can I change the API key after installation? - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+---
+
+In early 2021, we have changed how our login system works for the MultiSafepay Payment app for Lightspeed eCom. Our old system was password based, although now our new system uses the Single Sign On provided by Lightspeed. This means you can directly login via Lightspeed eCom store to our settings page of the app.
```
Comment (Kris-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/how-to-login.md:7]:
```suggestion
In early 2021, we have changed the login method for the MultiSafepay Payment app for Lightspeed eCom. The old login method was password-based, while the new method uses the single sign-on provided by Lightspeed. This means you can directly log in via the Lightspeed eCom store to our settings page of the app.
```
login = noun
log in =  verb

File: content/integrations/hosted/lightspeed_app/faq/how-to-login.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+To login using the new system you have to use our app store page. Login to the /admin section of your Lightspeed store.
```
Comment (Kris-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/how-to-login.md:9]:
```suggestion
To log in using the new method you have to use our app store page. Login to the /admin section of your Lightspeed store. 
```

**Review by Kris-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/integrations/hosted/lightspeed_app/faq/how-to-login.md (hunk 4 above)
Comment (Lars-MultiSafepay) [reply content/integrations/hosted/lightspeed_app/faq/how-to-login.md]: 👍 ty

**Review by Jasper-MultiSafepay — APPROVED** [review]

**Pull request merged**